src/Data/Gdpr/GdprDataClass.ts

```diff
@@ -9,6 +10,86 @@ export const Gdpr = provideDataClass(
       return (await import('./gdprParamsFallback')).gdprParamsFallback()
     }
 
+    if (!isUserLoggedIn()) {
+      const Authorization = getTokenAuthorization()
+      if (Authorization) {
+        const { url, headers: baseHeaders } = restApi
+        const headers = { ...baseHeaders, Authorization }
+
+        // TODO: Return { url, headers }, once #11616 is resolved
```

Comment:
* [ ] FYI: If I understand this comment correctly we would also drop the `does not support` checks. If this is correct, how about removing them right now?

Reply:
One #11616 is available `filters`, `order` and `search` is supported by the backend. But since I don't want to implement them in the "manual fetch" way I opted for `does not support` messages.